package.json

```diff
@@ -40,9 +42,13 @@
     "sanitize-html": "^2.10.0",
     "tailwind-merge": "^1.12.0",
     "tailwindcss": "^3.3.1",
-    "wagmi": "^0.12.7"
+    "usehooks-ts": "^2.9.1",
+    "viem": "^1.10.12",
```

Comment:
this is replacement for `ethers.js`, quite a lot of breaking changes

Reply:
Yes though we already had wagmi in the project. What I did here is upgrade wagmi and it was used in very few places.